rcv/combine_modules.py

```diff
@@ -0,0 +1,46 @@
+import cv2
+import json
+from find_parameters_by_coordinates import find_distance_and_angle_by_coordinates

```

Comment:
Module find_parameters_by_coordinates does not contain function  find_distance_and_angle_by_coordinates